spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/FindIncreasedAccessibilityOfMethodsTest.java

```diff
@@ -0,0 +1,122 @@
+package edu.umd.cs.findbugs.detect;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+import org.junit.Test;
```

Comment:
The project got migrated from junit to junit jupiter. Can you please change the import to `org.junit.jupiter.api.Test`? After merging to master, it shouldn't cause any issues, since all the other configs are taken care of.